dex/networks/zec/block_test.go

```diff
@@ -0,0 +1,191 @@
+		"cc3aeb56737a93cb9c1999d2bc44ada8e991d4193f125b575a722f3f557616e21d88ce" +
+		"6436f7eaaea4dfecad98893b2cf4e5e126c956a8f80ba8cb33596379a8e76e5126c08b" +
+		"38c5013bc5c9cdd1becbac6a90abc2a3bab9b5d854534507f32ef225261d313a2c70f8" +
+		"f421a1606bf2d978c808e8b7422bd935a0cc9e130174dfecd4fc526b2d5561cb7fe009" +
+		"28efc1b91bc702b44ec4ddf7d311a3140b4a5cb24a97f0a8da7b0879ab588d422dcfbe" +
+		"53a6407daec0a2da4b1f45f465886247375d92280ffc6821dd933e3490766ab63f031a" +
+		"187ecbd96ab0cded393c50e644ec157504f61763d3d6aed65ee6bfbd1e1f046120d2eb" +
+		"1143b8db4a9259f14071c111abaee41f2d6bfa315fa9b9fb946974641d5815ba9fc935" +
+		"8db908fa0fba4f44145a80eaa227c73809bace4a37ea76426ad6235a567c3f403ff445" +
+		"05316e1df97171f8f20a1421c599a5a407996343fd8be00337ab6b3e89ea3a5afe8d6b" +
+		"1dbded5a37066ea541803f8fccbe63731f72c9c19f08070078b05428735d1ffabb595f" +
+		"72a836bfe1c6c251b814defb7380b267884d543e1a780700f3f57c724e3e18f2e3090d" +
+		"f4f452b65ab2b636ff172f1d2fefd06e45a8e91177d857bc34fe5b9cb3453a00bbdab3" +
+		"efc9e5cbb1d53abe55180d7ac95a221c49c197503618b442f574a625b071c9aa5a95df" +
+		"91b21fae05bba68fc9b5cab924568f36e031de2c77b30a31d6961de61a415b32fc44de" +
+		"31de5622859f5a1d1f2781097454cd0424c1cd63fca937b73e4e15f6cca829920b4d57" +
+		"a6a4416f7e743d583e3b677dfebedde506289b58400d8730a85654c53652810a36ea73" +
+		"f1110eae15f8304534b7bb5fe163ed94ec495a514f6ad34cc98f3472e5f3f5e81af7e7" +
+		"90f7ae5a91cbbc7a2d5ae6f38c85e08367873cd622534b1ebb690adfaf470bc28ad9cf" +
+		"038f476506530bd1a80be599b05b8a7e5486d6443da4402daa0e372b7c501f8a4fdef3" +
+		"25082e7936ac6714426fb334c5c9c24bc071eae5f907e34701b424a12cef39b686e50c" +
+		"0ad86baa5ad6b98a34013e13dfe8df895dbeb5d2235bf48965055919350e0bbb6fd2cf" +
+		"4ad72b8a3e712f8a4bd7516f652d0b2702d652a7a2eb7b0ddaabe0b06d44ff49f186b2" +
+		"afa911a0b98b108558d839a0019d529ddb0574644ea7ee0ce16944dc1033e3638402eb" +
+		"64712dd6467a92aee70d97f930a3500fd39e67f19fee7caecbf4047fda681942f1724d" +
+		"f387c9c63426aebc6cde8a92375cec6d5cbd0cf68332be45ac3da7181ec77b96895937" +
+		"c757023f70457acef3869330827a1f6511d15fb5176b1d0d81a75c03d2fed0bc1b713c" +
+		"c8f12f95c56c5aad69076e0ab81876cb47dcf143011ad20009ddfd9390c8215f019a06" +
+		"9d3ac93ace45d86fca9a198913d8720805e4d77b0782a7e355ae4067856e0b7d7e462f" +
+		"7ba21517fc89e5561adb2129bbc8737d786f0f90df606a154955b845889283b74704ce" +
+		"c758fbce1bc9713c2f170569962567e27dcdf7f5f4bdee1ac2c3d877988e0104000080" +
+		"85202f8901000000000000000000000000000000000000000000000000000000000000" +
+		"0000ffffffff210387c9181c4d696e656420627920416e74506f6f6c36323361001f03" +
+		"20a3c16b65ffffffff0438c94d010000000017a914df37da178caa3ba0cc7feea71cd8" +
+		"63c6677f4ea587286bee000000000017a914d45cb1adffb5215a42720532a076f02c7c" +
+		"778c908740787d010000000017a914931fec54c1fea86e574462cc32013f5400b89129" +
+		"8780b2e60e000000001976a9145c38e5e20b62bbb5683dd68677ac3715047341ea88ac" +
+		"00000000000000000000000000000000000000")
```

Comment:
I'd have crack at `go:embed` for the blocks.  See my previous doge block deserializer PR and the current ltc 0.21 pr https://github.com/decred/dcrdex/pull/1536

You can write straight into a .dat file suitable for a `[]byte` var using `xxd`:
`./zcash-cli getblock <hash> 0 | xxd -p -r - blockX.dat`